interface-spec.md

```diff
@@ -192,6 +191,6 @@ Access to stores LDP or SPARQL endpoints can be implemented with a Store intefac
   All quads matching the pattern will be removed.
   The `end` and `error` events are used like described in the `Stream` interface.
 
-- `EventEmitter .deleteGraph(IRI|string graph)`
+- `EventEmitter .deleteGraph(Term|string graph)`
```

Comment:
can this one use more specific NamedNode instead of Term? or we also want to use it to delete the DefaultGraph?